src/java/org/apache/cassandra/streaming/OutgoingStream.java

```diff
@@ -47,7 +47,12 @@
     UUID getPendingRepair();
 
     String getName();
-    long getSize();
+
+    /**
+     * @return estimated file size to be streamed. This should only be used for metrics only, because concurrent
```

Comment:
grammar pedant: Could get rid of the second/redundant "only"

Reply:
+1